test/plugin/scenarios/dubbo-2.7.x-scenario/src/main/java/org/apache/skywalking/apm/testcase/dubbo/CaseServlet.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.apm.testcase.dubbo;
+
+import org.apache.skywalking.apm.testcase.dubbo.services.GreetService;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+/**
+ * @author zhangwei
```

Comment:
@arugal I think @kezhenxu94 's meaning is, you shoudl put @ascrutae 's name and your name on it. Rather than removing it.

Reply:
> @arugal I think @kezhenxu94 's meaning is, you shoudl put @ascrutae 's name and your name on it. Rather than removing it.

ok, I'll do that when I modify the jvm-container

Reply:
Sure. I am waiting @dmsoldmto confirm whether jvm container could be faster. Because the fat kar packaging stage could take more time.

Anyway, we will have the recommendation soon.